docs/superpowers/specs/2026-05-08-optimistic-update-cache-layer-design.md

```
@@ -0,0 +1,320 @@
+3. Riverpod NotifierProvider 기반 캐시 레이어를 도입하되, 영향 범위는 본 이슈 3개 도메인으로 한정. 앱 전체 상태관리 마이그레이션은 별도 백로그.
+
+## 3. 비목표
+
+- 앱 전체 Riverpod 마이그레이션 (별도 이슈)
+- Item 전체 필드 캐시화 (좋아요 관련 필드만 우선)
+- Undo 토스트 / 5초 되돌리기 UI (별도 백로그)
+- 새로운 외부 패키지 추가(freezed 등). 내부망 환경 제약상 기존 의존성만 사용.
+
+## 4. 아키텍처
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)